packages/avatar/contracts/nft-collection/NFTCollection.sol

```diff
@@ -102,6 +80,10 @@ contract NFTCollection is
          * @notice stores the personalization mask for a tokenId
          */
         mapping(uint256 => uint256) personalizationTraits;
+        /**
+         * @notice stores the number of tokens minted by an address
```

Comment:
minted by an address or minted for an address?

Reply:
"minted by" is the correct one